.github/workflows/e2e-tests.yml

```diff
@@ -154,3 +154,72 @@ jobs:
       - name: Install and Run Integration Tests
         run: |
           ./mvnw ${MAVEN_ARGS} -Pitests -pl $IT_MODULE verify
+  openshift-kubernetes-distribution:
+    name: CRC ${{ matrix.crc }} / OKD ${{ matrix.okd }}
+    needs: buildWithoutTests
+    runs-on: ubuntu-latest
+    if: github.repository == 'fabric8io/kubernetes-client'
+    env:
+      SHELL: /bin/bash
+    strategy:
+      fail-fast: false
+      matrix:
+        # OKD version is bound to crc version
+        okd: [v4.14.0]
+        crc: [2.32.0]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Cache .m2 registry
+        uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          key: cache-e2e-${{ github.sha }}-${{ github.run_id }}
+      - name: Setup Java 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Install required virtualization software
+        run: |
+          sudo apt-get update
+          sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system
+          sudo usermod -a -G libvirt $USER
+      - name: Remove unwanted stuff to free up disk image
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
+
+          sudo docker image prune --all --force
+
+          sudo swapoff -a
+          sudo rm -f /mnt/swapfile
+          sudo adduser $USER libvirt
+          sudo adduser $USER kvm
+          sudo usermod -a -G libvirt $USER
```

Comment:
Have you tried not to include this extra-step?

Reply:
When I try to remove this step in my [test repository github action](https://github.com/rohankanojia-forks/crc-okd-4-15-github-action-poc/actions/runs/11178021397/job/31074742305), I see this error:
```
write /home/runner/.crc/cache/tmp-extract/crc_okd_libvirt_4.14.0-0.okd-scos-2024-01-10-151818_amd64/crc.qcow2: no space left on device
Error: Process completed with exit code 1.
```